### PR TITLE
Add Python Resources Page

### PR DESCRIPTION
**Pull request by koustuvsinha**
As per discussion with @alayek, we found that most of the newcomers in Python Gitter channel are requesting resources and challenges to practice Python until the FCC challenges are up. Therefore I have curated a list of un-opinionated Python tutorials and Challenges for the newbies to try out.

Requesting @QuincyLarson to review. Since these links are external so please review them as to which ones should be added or which one shouldn't be. If needed I am happy to add more links.

Tagging @Rafase282

### CODE REVIEW

File: Python-Resources.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Python Resources
+
+[Home](Python)
+
+## Tutorials
+
+* [Google Python Class](https://developers.google.com/edu/python/)
+* [Python Official Tutorials](https://docs.python.org/3/tutorial/)
```
Comment (alayek) [inline Python-Resources.md]: Python official tutorial should be on top.

File: Python-Resources.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* [Learn Python the Hard Way](http://learnpythonthehardway.org/book/)
+* [Think Python](http://www.greenteapress.com/thinkpython/html/index.html)
+* [Learn Python in X minutes](https://learnxinyminutes.com/docs/python/)
+* [Learn Python 3 in X minutes](https://learnxinyminutes.com/docs/python3/)
+* [Dive into Python 3](http://www.diveintopython3.net/)
+* [The Hitchhiker's Guide to Python](https://python-guide.readthedocs.org/en/latest/)
```
Comment (alayek) [inline Python-Resources.md]: This is too good a document to be this down. I would recommend putting it second.

File: Python-Resources.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+* [Full Stack Python](http://www.fullstackpython.com/)
+* [Automate the Boring Stuff with Python](https://automatetheboringstuff.com/)
+
+## Challenges
+
+* [Exercism Python Challenges](http://exercism.io/languages/python)
+* [Python Koans](https://github.com/gregmalcolm/python_koans)
```
Comment (alayek) [inline Python-Resources.md]: Put Python Koans above Exercism.

**Comment by alayek** [comment]
@QuincyLarson please check if we need to remove any of these, for legal reasons.

**Comment by alayek** [comment]
@ranjanashish please see if you can add some of your resources. The focus is mostly on practicing Python. Tutorials and challenges.

File: Python-Resources.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* [Exercism Python Challenges](http://exercism.io/languages/python)
+* [CodingBat Python Challenges](http://codingbat.com/python)
+* [Learn Python Interactively](http://www.learnpython.org/)
+* [Project Euler](http://projecteuler.net/)
+* [Rosalind Python Bio-informatics Problems](http://rosalind.info/problems/locations/)
+* [Python Elevator Challenge](https://github.com/mshang/python-elevator-challenge)
+
+## Community
+
```
Comment (ranjanashish) [inline Python-Resources.md]: [Planet Python](http://planetpython.org/) can also be added here.

**Comment by Rafase282** [comment]
Looks good to me but the legal part I'll leave to @QuincyLarson

File: Python-Resources.md (hunk 4 above)
Comment (koustuvsinha) [inline Python-Resources.md]: added below

**Comment by Rafase282** [comment]
I'm just waiting for Quincy to okay this with regards to the legal aspects as you asked.

**Comment by alayek** [comment]
@Rafase282 let's wait for @QuincyLarson to okay this. No rush.

**Comment by QuincyLarson** [comment]
@koustuvsinha great idea! Are all of these resources free? I loved Python Koans.

**Comment by QuincyLarson** [comment]
There shouldn't be any legal concerns here - we are happy to link to solid external resources, and have done that throughout our community's history (most recently: https://medium.freecodecamp.com/25-free-resources-for-new-javascript-developers-11342cf1f4df#.tuc8r5zav)

**Comment by abhisekp** [comment]
- https://repl.it/languages/python3 for Python REPL for sharing code snippet
- http://pythontutor.com/ for Visual Python code walkthrough

**Comment by koustuvsinha** [comment]
@abhisekp added the links!
@Rafase282 consider merging?

**Comment by koustuvsinha** [comment]
@QuincyLarson all the above resources are free! Good idea for creating a similar medium article for python :)

**Comment by abhisekp** [comment]
Would be nice to categorize in the format in [**`JS Resources`**](https://github.com/FreeCodeCamp/wiki/blob/master/JS-Resources.md) with pretty links and icons.

**Pull request merged**